src/Keboola/S3Extractor/S3AsyncDownloader.php

```diff
@@ -28,7 +28,8 @@ class S3AsyncDownloader
     //    The SDK downloads the response body to a PHP temp stream by default.
     //    This means that the data stays in memory until the size of the body reaches ***2 MB***,
     //    at which point the data is written to a temporary file on disk.
-    private const MAX_CONCURRENT_DOWNLOADS = 100;
+    // 100 is too much for large files.
+    private const MAX_CONCURRENT_DOWNLOADS = 50;
```

Comment:
Z nejakeho dovodu bolo `100` este prilis vysoko, asi je tam este najaka rezia naviac k tym `2MB`.
Teraz [job uz bezi](https://connection.keboola.com/admin/projects/9376/queue/979828032) ....

Reply:
Ak by bol v buducnosti este problem, ta sa to moze este znizit alebo sa moze mierne zvysit memory limit.